share/functions/__fish_git_prompt.fish

```diff
@@ -367,7 +367,7 @@ function __fish_git_prompt --description "Prompt function for Git"
         else
             if test -n "$__fish_git_prompt_showdirtystate"
                 set -l config (command git config --bool bash.showDirtyState)
-                if test "$config" != "false"
+                if test "$config" != false
```

Comment:
The script is inconsistent about its use of quotes. No objection to cleaning that up in a separate PR but it's bad form to mix style cleanups with functionality changes; especially since you're not really making the script consistent.